docs/source-fabric/conf.py

```diff
@@ -271,6 +272,10 @@
     # These seem to be missing in reference generated API
     ("py:class", "torch.distributed.fsdp.wrap.ModuleWrapPolicy"),
     ("py:class", "torch.distributed.fsdp.sharded_grad_scaler.ShardedGradScaler"),
+    # Mocked optional packages
```

Comment:
This means that docs won't have link to these instances, I think it good to have especially in docs...

Reply:
They never had links to these instances because we don't install these dependencies in the docs build. The docs how they look and behave stays the same as before this PR, no changes!

Reply:
but these re sourced from `intersphinx_mapping`, not from installed packages so either ignore or add but does make sense to add mapping and then still ignore them...

Reply:
These dependencies can't be resolved. Here is the error output from the docs build without these ignores: https://github.com/Lightning-AI/lightning/actions/runs/6201757925/job/16839100845?pr=18561

So I'm keeping it for now.